README.md

```diff
@@ -25,13 +22,11 @@ Please note that the documentation below is for `winston-loggly-bulk@2.x`. [Read
   winston.add(new Loggly({options}));
 ```
 
-The Loggly transport is based on [Nodejitsu's][2] [node-loggly][3] implementation of the [Loggly][0] API. If you haven't heard of Loggly before, you should probably read their [value proposition][4]. The Loggly transport takes the following options. Either 'inputToken' or 'inputName' is required:
+The Loggly transport is based on [Nodejitsu's][2] [node-loggly][3] implementation of the [Loggly][0] API. If you haven't heard of Loggly before, you should probably read their [value proposition][4]. The Loggly transport takes the following options.
 
-* __level:__ Level of messages that this transport should log.
 * __subdomain:__ The subdomain of your Loggly account. *[required]*
-* __auth__: The authentication information for your Loggly account. *[required with inputName]*
-* __inputName:__ The name of the input this instance should log to.
-* __inputToken:__ The input token of the input this instance should log to.
+* __token:__ The access token (previously inputToken) *[required]*
+* __auth__: The authentication information for your Loggly account.
```

Comment:
This is normally not needed right? Actually is basic auth supported at all anymore? I recommend testing it and if it doesn't work remove this.

Reply:
Again the package uses node-loggly-bulk and there are both token and auth. Token is required, auth seems to be useless, but still I wasn't sure whether it's save to remove it completely.